### PR TITLE
Development

### PR DESCRIPTION
**Pull request by GoktugYalcin**
Turned all structure to a class

### CODE REVIEW

File: ResumeTerminal.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        exitOnUnexpectedKey: true,
+        continueOnSubmit: true,
+        selectedStyle: this.#tk.bgBlue,
+        submittedStyle: this.#tk.bold.bgYellow,
+      });
+    
+      this.#singleColumnMenu.on("submit", this.singleColumnMenuSettings());
+    };
+  
+    singleColumnMenuSettings = () => async (arg) => {
```
Comment (mustafaemreozmen) [inline ResumeTerminal.js]: After new class structure, we don't need IIFE at singleColumnMenuSettings.

Comment (mustafaemreozmen) [inline ResumeTerminal.js]: We don't need async because that method is sync-worker.

Comment (mustafaemreozmen) [inline ResumeTerminal.js]: We must set this method as private because this is a callback method. We mustn't call from all instances.

**Review by mustafaemreozmen — CHANGES_REQUESTED** [review]
Some small change requests :)

File: ResumeTerminal.js (hunk 1 above)
Comment (GoktugYalcin) [reply ResumeTerminal.js]: Fixed 🥲

Comment (GoktugYalcin) [reply ResumeTerminal.js]: Fixed 🥲

Comment (GoktugYalcin) [reply ResumeTerminal.js]: Fixed 🥲

**Comment by mustafaemreozmen** [comment]
Turn wrapper.js from a procedural to an object-oriented approach.

**Pull request merged**